### PR TITLE
[DC-3045] Update retraction validation notebooks to use time travel snapshots

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

**Review by nishanthpp93 — COMMENTED** [review]
Looks good! However I would suggest adding it as a new script so we have both to use as needed.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Comment by hiro-mishima** [comment]
Hi @lledezma ,
Thanks for our offline catchup yesterday and for updating the scripts accordingly!
I would like to suggest adding a bit more comments to each of the scripts for clarity.

For `ehr_retraction_in_place_validation.py`:
```
This notebook validates in-place `only_ehr` retraction for the following datasets. It assumes time travel is enabled for them:
- `only_ehr`: combined/ combined release datasets
- `only_ehr`: CT deid base/ clean datasets
- `only_ehr`: RT deid base/ clean datasets
This notebook does NOT validate the following datasets:
- `only_ehr`: EHR datasets
- `only_ehr`: Unioned EHR datasets
- `rdr_and_ehr`: All datasets
- For the datasets above, use `retraction_in_place_validation.py` instead.
- `only_ehr`: RDR datasets (If `only_ehr`, it skips retraction for RDR)
This notebook is not tested against the following datasets. Use it carefully:
- `only_ehr`: Backup datasets
- `only_ehr`: Sandbox datasets
```

For `retraction_in_place_validation.py`:
```
This notebook validates in-place retraction for the following datasets.  It assumes time travel is enabled for them:
- `only_ehr`: EHR datasets
- `only_ehr`: Unioned EHR datasets
- `rdr_and_ehr`: RDR datasets
- `rdr_and_ehr`: combined/ combined release datasets
- `rdr_and_ehr`: CT deid base/ clean datasets
- `rdr_and_ehr`: RT deid base/ clean datasets
This notebook does NOT validate the following datasets:
- `only_ehr`: All datasets except EHR and Unioned EHR datasets
- For the datasets above, use `ehr_retraction_in_place_validation.py` instead.
This notebook is not tested against the following datasets. Use it carefully:
- `rdr_and_ehr`: Backup datasets
- `rdr_and_ehr`: Sandbox datasets
```

File: data_steward/analytics/cdr_ops/ehr_retraction_validation.py [hunk 1]
```diff
@@ -251,4 +251,4 @@
                                            mapping_table=provenance_table_for(
                                                table, is_deidentified)))
 mapping_check = '\nUNION ALL\n'.join(mapping_queries_list)
-execute(client, mapping_check)
+execute(client, mapping_check)
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/ehr_retraction_validation.py:254]: Minor change, but you can add the newline back here.

**Review by nishanthpp93 — APPROVED** [review]
I think this is good for now, we can do the EHR dataset validation later since it has 2k+ tables we only have a few of those.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**